src/res_send.c

```diff
@@ -1194,6 +1194,7 @@ send_dg(res_state statp,
 		if (have_sendmmsg >= 0 && nwritten == 0 && buf2 != NULL
 		    && !single_request)
 		  {
+#if 0
```

Comment:
Why not just remove this code?

Reply:
OK, I guess there's no good reason not to.
We always have previous versions to look at, if we need to.